src/store/Disks.cc

```diff
@@ -165,9 +179,7 @@ Store::Disks::store(int const x) const
 SwapDir &
 Store::Disks::Dir(const int i)
 {
-    SwapDir *sd = INDEXSD(i);
-    assert(sd);
-    return *sd;
+    return SwapDirByIndex(i);
```

Comment:
Please add two asserts to make sure `i` is within the valid range. Most of the existing callers do not need that assert, but I see at least one special case where the cache_dir index is computed (rather than simply iterated) and should be double checked.